docs/Visual-Studio-Extensibility/Visual-Studio-Templates.md

```diff
@@ -139,7 +139,7 @@ SDKs that are installed using an MSI can install NuGet packages directly on the
 
 1. Write a registry key to identify the package location:
 
-    - Key location: `HKEY_LOCAL_MACHINE\SOFTWARE[\Wow6432Node]\NuGet\Repository`
+    - Key location: Either the machine-wide `HKEY_LOCAL_MACHINE\SOFTWARE[\Wow6432Node]\NuGet\Repository` or if it's per-user installed templates and packages, alternatively use `HKEY_CURRENT_USER\SOFTWARE\NuGet\Repository`
```

Comment:
No didn't work. Also if setting this from the vsix it doesn't go in the wow node either. Don't think wow applies to the user part at all. It's more or less empty